internal/cmd/run/playwright.go

```diff
@@ -154,10 +156,16 @@ func runPlaywrightInSauce(p playwright.Project, regio region.Region, tc testcomp
 			ShowConsoleLog:     p.ShowConsoleLog,
 			ArtifactDownloader: &rs,
 			Reporters:          createReporters(p.Reporters),
+			SlackService:       &tc,
+			Notifier: &slack.Notifier{
+				Channels:  p.Notifications.Slack.Channels,
+				Framework: "playwright",
+				Metadata:  p.Sauce.Metadata,
+				TestEnv:   "sauce",
+			},
 		},
 	}
 
-	cleanPlaywrightPackages(&p)
```

Comment:
Is that removal expected ?

Reply:
Thanks! It was removed unintentionally during rebase with the main branch. I returned it back.